napari/components/tests/test_multichannel.py

```diff
@@ -1,7 +1,8 @@
 import numpy as np
 import dask.array as da
 from napari.components import ViewerModel
-from napari.utils import colormaps
+from napari.utils.colormaps import colormaps
```

Comment:
why this change?

Reply:
I added a couple of source files and tests to that module, and I thought it would be better to hold them in a separate module. I think this change was made in parallel in a different PR as well.